packages/twitch-pubsub-client/src/SingleUserPubSubClient.ts

```diff
@@ -194,7 +219,7 @@ export class SingleUserPubSubClient {
 				return new PubSubWhisperMessage(messageData as PubSubWhisperMessageData);
 			}
 			default:
-				return undefined;
+				return new PubSubUndocumentedMessage(messageData as PubSubUndocumentedMessageData);
```

Comment:
Now that this can't return undefined anymore, please remove `undefined` from the return type of this method. This in turn means that you should also remove the condition `if (message)` in its caller.